bedrock/products/urls.py

```diff
@@ -25,6 +25,51 @@
     path("vpn/mobile/ios/", views.VPNIosView.as_view(), name="products.vpn.platforms.ios"),
     path("vpn/mobile/android/", views.VPNAndroidView.as_view(), name="products.vpn.platforms.android"),
     page("vpn/ipad/", "products/vpn/platforms/ipad.html", ftl_files=["products/vpn/platforms/ipad", "products/vpn/shared"]),
+    # Evergreen SEO articles (issue #10224)
+    path(
```

Comment:
You could compress this into one URL conf and a mapping of slug to old template and ftl files in the views.py file. 

```python
path("vpn/more/<slug:slug>/", views.vpn_resource_center_redirect, name="products.vpn.more.redirect")
```
Then in views.py
```python
VPNRC_SLUGS = {
    "what-is-an-ip-address": {
        "slug": "what-is-an-ip-address",
        "old_template": "products/vpn/more/ip-address.html",
        "ftl_files": ["products/vpn/more/ip-address", "products/vpn/shared"],
    },
    "vpn-or-proxy": {
        "slug": "the-difference-between-a-vpn-and-a-web-proxy",
        "old_template": "products/vpn/more/vpn-or-proxy.html",
        "ftl_files": ["products/vpn/more/vpn-or-proxy", "products/vpn/shared"],
    },
}
```
You've already done the work here, so you don't really need to do this, but for future reference that's how I'd probably do it. This works just fine though.

Reply:
This is great, thanks!